requirements/edx/base.txt

```diff
@@ -164,7 +164,9 @@ nodeenv==1.5.0            # via -r requirements/edx/base.in
 numpy==1.18.5             # via -c requirements/edx/../constraints.txt, chem, openedx-calc, scipy
 oauthlib==3.0.1           # via -c requirements/edx/../constraints.txt, -r requirements/edx/base.in, django-oauth-toolkit, lti-consumer-xblock, requests-oauthlib, social-auth-core
 openedx-calc==1.0.9       # via -r requirements/edx/base.in
-ora2==2.12.0              # via -r requirements/edx/base.in
+# FIXME: revert when PR merged
+# cf https://github.com/edx/edx-ora2/pull/1350
+git+https://github.com/open-craft/edx-ora2.git@jill/ora-assessment-report#egg=ora2==2.13
```

Comment:
Once you've made the other change and tested it manually with ORA, or whatever you'd like, can you please remove this? Then I'll approve the PR and merge.

Reply:
Done! I also merged latest master and updated the sandbox, and was able to download a new example CSV file (linked in description), so you can test it there if you like.